docs/source/onedal/algorithms/clustering/kmeans.rst

```diff
@@ -120,6 +126,12 @@ Programming Interface
 
 Refer to :ref:`API Reference: K-Means <api_kmeans>`.
 
+----------------
+Distributed mode
+----------------
+
+The algorithm supports distributed execution in SPMD mode (only on GPU).
```

Comment:
```suggestion
The algorithm supports distributed execution in the SPMD mode and can be run on a GPU only.
```

Reply:
Moreover, I have copied this sentence from other section which has been approved. It is necessary to change there too if something is incorrect

Reply:
I don't think so. The bracketed statement applies to the sentence before it and this sentence declares only that the algorithm can be run in distributed SPMD mode (as far as I understand distributed and SPMD are synonyms in this context). So, the only conclusion can be drawn is that the distributed mode is supported only for GPU.

Your last suggestion is also correct, and if we are trying to decrease brackets number in text or something like that then I can push your version :) But as for me, the current version looks more briefly and neatly.